decision-trees.Rmd

```diff
@@ -411,6 +409,8 @@ dt_tune_metrics %>%
   arrange(desc(mean))
 ```
 
+<!-- BC: I'm seeing 40 as the best minimum n. -->
+
 Unsurprisingly, 30 is looking best for our minimum $n$. To be sure we've got this right though, let's set cost_complexity and tune *just* on the minimum $n$. We know that values of 21 and 40 are both worst than 30, but let's see if there's any more room for optimization around there. 
```

Comment:
I think there might be an issue with reproducibility -- perhaps we need to set a seed? This is another example where I'm getting a different interpretation based on the output from what you originally wrote. Sorry if I'm just misunderstanding something!

![Screen Shot 2020-12-30 at 1 34 31 PM](https://user-images.githubusercontent.com/32279312/103383212-c4b2a280-4aa6-11eb-97c4-cbc4b2f86748.png)

Reply:
Ugh... lame. Yeah I'll work on this.